backend/src/main/java/gov/cdc/usds/simplereport/api/model/accountrequest/AccountRequest.java

```diff
@@ -16,20 +16,23 @@
   @NotNull private String email;
   @NotNull private String workPhoneNumber;
   private String cellPhoneNumber;
-  @NotNull private String mailingAddress1;
-  private String aptSuiteOther;
-  private String aptFloorSuiteNo;
+  @NotNull private String streetAddress1;
+  private String streetAddress2;
```

Comment:
Renaming existing fields that are `@NotNull` will break requests from the deployed site.  We can use Jackson to handle this, though.